File: js/04-heranca/Classes/Pessoa/Pessoa.js
class Pessoa{
    constructor(id, mainName, fone, city){
        this.id = id;
        this.mainName = mainName;
        this.fone = fone;
        this.city = city;
    }

    imprimir(){
        console.log(`${this.id}, ${this.mainName}, ${this.fone}, ${this.city.name}`);
    }
}        

export { Pessoa }

File: py/04-heranca/Classes/Pessoa/Pessoa.py
from Classes.Cidade.Cidade import Cidade

class Pessoa:

    def __init__(self, id=None, name=None, fone=None, city=Cidade(None, None)):
        self.id = id
        self.name = name
        self.fone = fone
        self.city = city

    def imprimir(self):
        print(self.id, self.name, self.fone, self.city.name)